python/entry_point.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+source "${DCOS_SERVICE_DIAGNOSTICS_DIRECTORY}"/init_container.sh
```

Comment:
Can we get rid of this file by invoking `init_container.sh` directly in `ENTRYPOINT`?

Reply:
I have introduced the script `entry_point.sh` intentionally to invoke `init_container.sh` relative  to DCOS_SERVICE_DIAGNOSTICS_DIRECTORY env var. It's especially helpful in debug mode for tracing and debugging that one without container restarts. 
Dockefile ENTRYPOINT exec form of syntax unfortunately is not evaluate env vars, so I can't just write  `ENTRYPOINT ["${DCOS_SERVICE_DIAGNOSTICS_DIRECTORY}"/init_container.sh]` - it won't work.

Reply:
Ah, OK, please add a comment to this script saying so.

Reply:
@VladimirKravtsov what about setting `WORKDIR` to that environment variable?

https://docs.docker.com/engine/reference/builder/#environment-replacement

Reply:
@mpereira it's perhaps make sense. But I'm not sure that ENTRYPOINT evaluates relatively WORKDIR. I will check

Reply:
It does:

![image](https://user-images.githubusercontent.com/112426/65425221-c7d3d300-de0d-11e9-96bd-61a065053186.png)

Reply:
I'm not sure that i correctly understood your general idea to set WORKDIR to DCOS_SERVICE_DIAGNOSTICS_DIRECTORY env variable. But I decided to playground with WORKDIR and use in instead DCOS_SERVICE_DIAGNOSTICS_DIRECTORY - and it works fine!
So, it takes omit  the python/entry_point.sh file at all and simplify `create_serve_diagnostics_bundle.sh` script. 
@porridge @mpereira PTAL